src/main/java/com/codurance/retropolis/entities/Column.java

```diff
@@ -1,32 +1,24 @@
 package com.codurance.retropolis.entities;
 
+import java.util.ArrayList;
 import java.util.List;
 
 public class Column {
 
   private Long id;
-  private Long boardId;
   private ColumnType title;
-  private List<Card> cards;
+  private List<Card> cards = new ArrayList<>();
```

Comment:
Please move that `= new ArrayList<>();` inside the body of the `public Column(Long id, ColumnType title){` constructor. It's a bad practice to instantiate like that.